libreantdb/api.py

```diff
@@ -40,9 +40,9 @@ class DB(object):
     this class contains every query method and every operation on the index
     '''
     # Setup {{{2
-    def __init__(self, es):
+    def __init__(self, es, index_name='book'):
```

Comment:
default index name should be ```libreant``` in order to be consistent with the default choosen in webant module and because we do not store only book!